behaviors/README.md

```diff
@@ -9,20 +9,137 @@ behaviors/
 
 ## Defined in the schema
 
-Behaviors are added to fields in your `schema.yaml`. They are an array of objects, with a `fn` property and any number of arguments:
+Behaviors are added to fields in your `schema.yaml`. They are an array of strings and/or objects (with a `fn` property and any number of arguments):
 
 ```yaml
-my-field:
+myField:
   _has:
-    -
-      fn: text
-      required: true
-      placeholder: This is my placeholder
+    - text
     -
       fn: soft-maxlength
```

Comment:
Maybe a comment showing that `fn: soft-maxlength` maps to `behaviors/soft-maxlength.js`? Not sure... might be repetitive but maybe would make it clearer.